### DISCUSSION TITLE
Reduce whitespace on first section of article

### DISCUSSION THREAD

**Discussion by papafe** [jpanther/congo#226] [Q&A] [answered]
There is some space a the top of the content of the article. If the first section of the article it's a paragraph then it's not so much:

<img width="1684" alt="p" src="https://user-images.githubusercontent.com/4850119/175281947-98b53e6e-bdca-4075-b82b-6ed4ddf82479.png">

But if it's an heading then it starts to become quite a lot and it looks like there's something missing in the page:

<img width="1691" alt="h2" src="https://user-images.githubusercontent.com/4850119/175281975-31ee8d4b-a2e3-4cc8-9b3a-283169ecfd13.png">

I would say that the space makes sense between paragraph/headings in the body of the article, but probably it should not be there on top of the first section. 
Is it possible to remove it somehow?

**Comment by jpanther** [comment, marked as answer]
Thanks for the suggestion. This will be fixed in the next release.

**Reply by papafe** [reply]
Glad to hear, thanks a lot!
